# Total planning attempts
max_solutions: 10

# Planning group and link names
arm_1_group_name: "panda_1_arm"
arm_2_group_name: "panda_2_arm"
eef_1_name: "hand_1"
eef_2_name: "hand_2"
hand_1_group_name: "hand_1"
hand_2_group_name: "hand_2"
hand_1_frame: "panda_1_link8"
hand_2_frame: "panda_2_link8"

# Poses
hand_open_pose: "open"
hand_close_pose: "close"
arm_home_pose: "ready"

# Scene frames
world_frame: "world"
table_reference_frame: "world"
object_reference_frame: "world"
surface_link: "table"

# Gripper grasp frame transform [x,y,z,r,p,y]
grasp_frame_transform: [0, 0, 0.1, 1.571, 0.785, 1.571]

# # Place pose [x,y,z,r,p,y]
# place_pose: [0.6, -0.15, 0, 0, 0, 0]
# place_surface_offset: 0.0001 # place offset from table

# Valid distance range when approaching an object for picking
approach_object_min_dist: 0.1
approach_object_max_dist: 0.15

# Valid height range when lifting an object after pick
lift_object_min_dist: 0.01
lift_object_max_dist: 0.1
